fcrepo-integration-rdf/src/test/java/org/fcrepo/integration/rdf/AbstractIntegrationRdfIT.java

```diff
@@ -125,7 +125,7 @@ private static Graph getTidiedGraph(final Graph graph) {
 
             Triple replacement = next;
 
-            if (replacement.getSubject().toString().contains(".well-known")) {
+            if (replacement.getSubject().toString().contains("#genid")) {
```

Comment:
No, more that you're just using `String::contains` on the whole URI. You could parse into `URI` and use the specific methods there.